ui/src/app/views/admin-review-id-verification/component.ts

```diff
@@ -0,0 +1,54 @@
+import {Component, OnInit} from '@angular/core';
+
+import {ErrorHandlingService} from 'app/services/error-handling.service';
+
+import {
+  IdVerificationReviewRequest,
+  Profile,
+  ProfileService,
+} from 'generated';
+
+
+/**
+ * Review ID Verifications. Users with the REVIEW_ID_VERIFICATION permission use this
+ * to manually set (approve/reject) the ID verification state of a user.
+ */
+@Component({
+  templateUrl: './component.html',
+  styleUrls: ['./component.css']
+})
+export class AdminReviewIdVerificationComponent implements OnInit {
+  profiles: Profile[] = [];
+  contentLoaded = false;
+
+  constructor(
+      private errorHandlingService: ErrorHandlingService,
+      private profileService: ProfileService
+  ) {}
+
+  ngOnInit(): void {
+    this.errorHandlingService.retryApi(this.profileService.getIdVerificationsForReview())
+        .subscribe(
+            profilesResp => {
+              for (const profile of profilesResp.profileList) {
+                this.profiles.push(profile);
+              }
+              this.contentLoaded = true;
+            });
+  }
+
+  approve(profile: Profile, approved: boolean): void {
+    const request = <IdVerificationReviewRequest>{
+      approved: approved,
```

Comment:
When a JS (or TS) object key is the same as the variable it references they may be collapsed, e.g. this line could be `const request = <IdVerificationReviewRequest>{approved};` and it would mean the same thing.  Using the abbreviated version is preferred.

Reply:
Also based on the workspace review component to keep things consistent, so I'm happy to follow a convention, but it seems like both should be the same.

Reply:
I think it's okay if newer code uses better conventions as we all learn the language better, but if consistency is important to you then I would vote to fix both here too.